.env

```diff
@@ -0,0 +1,2 @@
+GCP_ARTIFACT_REPOSITORY=bobanetwork-tools-artifacts
```

Comment:
One concern I have about this approach, is that even if we upstream these changes, we'll still have trouble getting CI to run on our PRs to upstream.  Basically, we need to ensure that these variables match the CircleCI instance it's executing in, not the particular repo/branch.  Is there an approach, maybe such as pushing these variables into the CircleCI config directly that would allow for CI to succeed in vanilla forks in another CircleCI instance?

Reply:
These are actually set in circleci config (as secrets) and not picked up from here. The purpose of this file here is that if you try building v3-anchorage locally, you would fetch our released artifacts. 

All builds in circleci should actually succeed without problems when we open a PR upstream, except things that write to designated Artifact Repository. 

Did I understand your question?

Reply:
Ah, perfect, thanks!